### ISSUE TITLE
QOL suggestion

### ISSUE THREAD

**Issue by tyler489** [TeamPneumatic/pnc-repressurized#143]
Make buckets of fluid that do not consume the bucket return the bucket to the inventory instead of leaving it empty bucket in crafting table..

**Comment by desht** [comment]
Are you referring to the crafting of speed upgrades with lubricant buckets? If so, see #139

**Comment by tyler489** [comment]
Yes. And i saw that. But was thinking that just returning a bucket to an inventory would also be nice. And probably easier to do

**Comment by tyler489** [comment]
Woops.. tried to scroll and hit close
